src/components/Error.tsx

```diff
@@ -0,0 +1,20 @@
+import { ErrorType } from '../utils/ErrorType';
+
+/* eslint-disable jsx-a11y/control-has-associated-label */
+type Props = {
+  setHasError: (value: ErrorType) => void,
+  setErrorMessage: () => string,
+};
+
+export const Error: React.FC<Props> = ({ setHasError, setErrorMessage }) => {
+  return (
+    <div className="notification is-danger is-light has-text-weight-normal">
+      <button
+        type="button"
+        className="delete"
+        onClick={() => setHasError(ErrorType.None)}
+      />
+      {setErrorMessage()}
```

Comment:
Consider using a variable instead of calling 'setErrorMessage()' directly in the JSX. This improves readability and allows for easier debugging.